src/cli/cli.ts

```diff
@@ -50,6 +50,9 @@ export class Cli {
         DB_REDIS_USERNAME: 'database.redis.username',
         DB_REDIS_PASSWORD: 'database.redis.password',
         DB_REDIS_KEY_PREFIX: 'database.redis.keyPrefix',
+        DB_REDIS_SENTINELS: 'database.redis.sentinels',
+        DB_REDIS_SENTINEL_PASSWORD: 'database.redis.sentinelPassword',
+        DB_REDIS_NAME: 'database.redis.name',
```

Comment:
To be dicussed: is this the optimal name for the environment variable?

In my opinion, `name` is not the best word to describe the option. It is the wording of `ioredis` though. My preferrence would be something like `DB_REDIS_INSTANCE_NAME` or `DB_REDIS_SERVICE_NAME`.

Because of the default value `mymaster`, I'm quite sure everyone which some Redis Sentinel background will know what the option is there for though. So this is some minor detail only.

Reply:
@Namoshek Both of them is a good name. Choose anyone. 😄

Reply:
`DB_REDIS_INSTANCE_NAME` it is then 👍